boot/zephyr/Kconfig

```diff
@@ -16,35 +16,6 @@ config MCUBOOT
 	select USE_DT_CODE_PARTITION if HAS_FLASH_LOAD_OFFSET
 	select MCUBOOT_BOOTUTIL_LIB
 
-config BOOT_USE_MIN_PARTITION_SIZE
```

Comment:
this is introduced in earlier commit.
Should we do a revert and then re-commit so that the patchset can be reduced in future history rewrite ?

Or make a comment in the commit message that it should be squashed with commit-x.

Reply:
I will identify the commits that added these and try to come up with a squashing-scheme

Reply:
I'm getting conflicts when reverting due to how the last upmerge was handled. IMO we could handle this by informing those responible for the next upmerge that none of the "noup" should have any modifications to `boot/zephyr/Kconfig` and that this commit should be removed at that point.